pkg/commands/execution_context.go

```diff
@@ -395,6 +397,8 @@ func (ctx *executionContext) pollEvents(c context.Context, osCh <-chan os.Signal
 		if ctx.cloudTestConfig.Statistics.Enabled {
 			ctx.printStatistics()
 		}
+	case err := <-ctx.terminationChannel:
```

Comment:
Can we reuse this channel in healthcheck to use only one channel with errors?

Reply:
The buffer size of healthCheck channel depends on number of check configs. Can we ensure that it is appropirate size for terminationChannel too? So, how we define buffer size for that combined channel?

Reply:
How we will distinguish errors from health check and wrap them with `errors.Wrapf(err, "health check probe failed")` as it is done now?

Reply:
> How we will distinguish errors from health check and wrap them with errors.Wrapf(err, "health check probe failed") as it is done now?

Can we just wrap the error in healthcheck goroutine?

Reply:
> Can we just wrap the error in healthcheck goroutine?
Of course!